src/Validation/Rules/ShortPhpLinesTrait.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Validation\Rules;
+
+use Closure;
+use Illuminate\Translation\PotentiallyTranslatedString;
+
+/**
+ * @internal
+ * @method int getInputLineLimit()
+ */
+trait ShortPhpLinesTrait
```

Comment:
Instead of trait with this `@method` magic mess, use normal service.

Reply:
sure, updated to use normal service https://github.com/rectorphp/getrector-com/pull/2490/commits/618cbd13acd50c252ecf8cf2d969621d1d9b4675